bool useGroundContactDetection
bool useGroundContactDetectionForLEDs
bool useGroundContactDetectionForSafeStates
bool useGroundContactDetectionForSensorCalibration
